service/history/historyEngine.go

```diff
@@ -1234,7 +1234,7 @@ func (e *historyEngineImpl) DescribeWorkflowExecution(
 			},
 			WorkflowExecutionTimeoutSeconds: executionInfo.WorkflowExecutionTimeout,
 			WorkflowRunTimeoutSeconds:       executionInfo.WorkflowRunTimeout,
-			WorkflowTaskTimeoutSeconds:      executionInfo.WorkflowTaskTimeout,
+			WorkflowTaskTimeoutSeconds:      executionInfo.DefaultWorkflowTaskTimeout,
```

Comment:
Maybe we should rename the DescribeWorkflowExecutionResponse to also call it `DefaultWorkflowTaskTimeoutSeconds` instead of `WorkflowTaskTimeoutSeconds` to stay consistent.

Reply:
There are bunch of `WorkflowTaskTimeoutSeconds` fields in proto. Let's discuss them separately since those changes should be started from `api` repo anyway.